test/e2e/apps/disruption.go

```diff
@@ -402,3 +403,18 @@ func waitForPdbToBeProcessed(cs kubernetes.Interface, ns string) {
 	})
 	framework.ExpectNoError(err, "Waiting for the pdb to be processed in namespace %s", ns)
 }
+
+func waitForPdbToObserveHealthyPods(cs kubernetes.Interface, ns string, healthyCount int32) {
+	ginkgo.By("Waiting for the pdb to observed all healthy pods")
+	err := wait.PollImmediate(framework.Poll, wait.ForeverTestTimeout, func() (bool, error) {
+		pdb, err := cs.PolicyV1beta1().PodDisruptionBudgets(ns).Get("foo", metav1.GetOptions{})
+		if err != nil {
+			return false, err
+		}
+		if pdb.Status.CurrentHealthy != healthyCount {
+			return false, nil
```

Comment:
How about adding `framework.Logf("current healthy count(%d) is different from expected healthy count(%d)", pdb.Status.CurrentHealthy, healthyCount)` for digging easily?
The log will be output maximum 15 times as current interval and timeout, but it would be fine if we want to debug.